content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss

```diff
@@ -11,6 +11,10 @@ $excerpt-line-height: 20;
 
   &:hover {
     header {
+      @include dark-theme-only {
+        color: $blue-40;
```

Comment:
I think I lean towards something Mardak might be saying here, just took me a bit to put it all together.

For this component's colour, we can just use `--newtab-link-primary-color` because in dark mode it's set to `$blue-40` and in light mode, it's at `$blue-60`

Reply:
I think we should decouple DS and AS as much as possible to prevent them from regressing each other if only one has changes.

Reply:
If we do use the current approach we should create a namespaced set of variables IMO (eg: `--ds-link-primary-color`)